keps/sig-cloud-provider/1959-service-lb-class-field/README.md

```diff
@@ -132,16 +139,18 @@ type ServiceSpec struct {
 	// implementation is watching for Services with a matching class name. Any default load balancer
 	// implementation (e.g. cloud providers) should ignore Services that set this field.
 	// +optional
-	LoadBalancerClassName string `json:"loadBalancerClass,omitempty"`
+	LoadBalancerClass string `json:"loadBalancerClass,omitempty"`
 }
 ```
 
 * `loadBalancerClass` will be immutable only when the Service type is `LoadBalancer`, this way existing and future implementations
-do not have to worry about handling Services that change the class name. The class name is mutable and can be cleared when the
+do not have to worry about handling Services that change the class name. The class name is mutable and must be cleared when the
 type changes.
 * `loadBalancerClass` will be validated against label-style format.
 * the `loadBalancerClass` field will be feature gated. The field will be dropped during API strategy unless
 the feature gate is enabled.
+* all external implementations of Service LoadBalancer using a non-empty class name should use the finalizer `service.kubernetes.io/load-balancer-cleanup`
+to ensure proper garbage collection of external resources.
 
 Required updates to service controller:
 * if the class field is NOT set for a Service, allow the cloud provider to reconcile the load balancer.
```

Comment:
line below (157) says "annotation", but rest of the KEP says it's a field

168s/propoerly/properly/

Reply:
Thanks, left-over when this was initially proposing annotations

Reply:
still have to fix line 168 (nit)